test/AspNetCoreModule.Test/Framework/InitializeTestMachine.cs

```diff
@@ -177,10 +296,14 @@ public InitializeTestMachine()
                     TestUtility.LogInformation("We will use IISExpress instead of IIS: " + ex.Message);
                 }
 
-                string siteRootPath = Path.Combine(Environment.ExpandEnvironmentVariables("%SystemDrive%") + @"\", "inetpub", "ANCMTest");
+                string siteRootPath = TestRootDirectory;
                 if (!Directory.Exists(siteRootPath))
                 {
                     Directory.CreateDirectory(siteRootPath);
+                    DirectorySecurity sec = Directory.GetAccessControl(siteRootPath);
```

Comment:
Will this be hit by running IIS express? If so, it requires an authenticated user, which I think needs Admin permissions?

Reply:
This does not require Admin permission. I already tested it.